sections/featured-blog.liquid

```diff
@@ -19,7 +19,7 @@
 
 {%- liquid
   assign posts_displayed = section.settings.blog.articles_count
-  if section.settings.post_limit <= section.settings.blog.articles_count
+  if section.settings.post_limit <= section.settings.blog.articles_count or section.settings.post_limit == 3
     assign posts_exceed_limit = true
```

Comment:
this was added to add the pagination to default blog post, from what I've seen it doesn't interfere with the other blogs, but would appreciate extra eyes on this to ensure it will not cause problems in the future.